samples/cellular/fmfu_smp_svr/README.rst

```diff
@@ -51,35 +51,58 @@ Building and running
 
 To use the UART overlay for increasing the transfer speed, add the ``-DDTC_OVERLAY_FILE=uart.overlay`` flag to your build.
 See :ref:`cmake_options` for instructions on how to add this flag.
-For example, when building on the command line, add the UART overlay for nRF9160 DK in the following way:
+When building on the command line add the UART overlay in the following way, depending on the DK you are using:
 
-  .. code-block:: console
+   .. tabs::
 
-     west build -b nrf9160dk_nrf9160_ns -- -DDTC_OVERLAY_FILE=uart.overlay
+      .. group-tab:: nRF9161 DK
+
+         .. code-block:: console
+
+            west build -b nrf9161dk_nrf9161_ns -- -DDTC_OVERLAY_FILE=uart.overlay
+
+      .. group-tab:: nRF9160 DK
+
+         .. code-block:: console
+
+            west build -b nrf9160dk_nrf9160_ns -- -DDTC_OVERLAY_FILE=uart.overlay
 
 Testing
 =======
 
 After programming the sample to your development kit, test it by performing the following steps:
 
-1. Connect the USB cable and power on or reset your nRF9160 DK.
+1. Connect the USB cable and power on or reset your nRF91 Series DK.
 #. Open a terminal emulator, observe that the sample starts, and then close the terminal emulator.
 #. Call the provided :file:`update_modem.py` script specifying the COM port, the firmware ZIP file, and the UART baud rate shown in the following examples.
 
    * If you used the default baud rate:
 
-     .. parsed-literal::
-        :class: highlight
+       .. group-tab:: nRF9161 DK
```

Comment:
Could have one group tab (after line no:77) for both the commands in instead of two.

Reply:
Also need to use .. tabs:: before the .. group-tab.

Reply:
I'm not sure how you mean. I've addressed the comments above. @divipillai  please add a commit on top if you want to change the group tab setup :)